interface/controller/app_controller_test.go

```diff
@@ -0,0 +1,55 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+)
+
+type Response struct {
+	Message string `json:"message"`
+}
+
+func TestHomeHandler(t *testing.T) {
+	w := httptest.NewRecorder()
+	r := httptest.NewRequest(http.MethodGet, "/", nil)
```

Comment:
Would it be good to have negative testcases? testing is open to your consideration :) just wondering, how much will you additionally cover.